Sources/Target/Windows/Process.cpp

```diff
@@ -27,6 +27,23 @@ using ds2::Utils::Stringify;
 
 #define super ds2::Target::ProcessBase
 
+#if defined(ARCH_ARM)
+static uint8_t const debugBreakCode[] = {
```

Comment:
Nit: I think global variables are marked with a `g` prefix in this code. e.g.: `gDebugBreakCode`.